internal/verifier/verifier_test.go

```diff
@@ -184,8 +258,55 @@ func TestVerifyBuilderImageDigest_DigestNotFound(t *testing.T) {
 	}
 }
 
-func TestVerifyRepoURI_FoundURI(t *testing.T) {
-	got := common.NewProvenanceIR(binarySHA256Digest,
+func TestVerify_NeedsButHasEmptyBuilderImageDigest(t *testing.T) {
+	builderImageSHA256Digest := ""
+	got := common.NewProvenanceIR(binarySHA256Digest, amber.AmberBuildTypeV1, common.WithBuilderImageSHA256Digest(builderImageSHA256Digest))
```

Comment:
```suggestion
	got := common.NewProvenanceIR(binarySHA256Digest, amber.AmberBuildTypeV1, common.WithBuilderImageSHA256Digest(""))
```

And remove line 262.

Reply:
Done.